README.md

```diff
@@ -1,95 +1,108 @@
-- **Comments**
-All public cslib functions should be self-documenting, i.e. contain \*comment lines before them that briefly describe: the purpose of the function, its arguments and its return value (if any).
+*label flavour_text_strength
 
+The highest stat is "strength"! Amazing! 
+```
 
-#### Naming
+### Modules
 
-Below follows guidance on naming variables and routines etc., when contributing code to cslib.
+|    Module    |               Description               | Version | Depends on |                          Routines                          |
+|:------------:|:---------------------------------------:|:-------:|:----------:|:----------------------------------------------------------:|
+| cslib_string | Routines to manipulate text (strings).  |  1.0.0  |            | concat, lowercase, substring, find, replace, reverse, etc. |
+| cslib_number |     Routines to manipulate numbers.     |  1.0.0  |            |                    mean, max_stat, max, min, etc.                    |
+|  cslib_menu  | Routines to generate menus and choices. |  1.0.0  |           |                                                            |
+|  cslib_loop  |   Repeat the execution of a routine.    |  1.0.0  |            |                           repeat                           |
 
-##### Functions
-Private functions should be prefixed with an underscore, e.g.: `*label _replace_helper`.
-Public functions, i.e. functions you intend a user (or another cslib scene) to call with \*gosub_scene, should be named as simply and intuitively as possible. They should **not** be prefixed with an underscore, e.g.: `*label replace`. Try to avoid special characters or numbers (without good reason). Words should be separated by an underscore.
-##### Parameters
-Named parameters (created via \*params) should similarly be free of special characters and numbers. They should also be prefixed with p_, e.g.: `*params p_start p_end`.
-Automatic `param_N` variables are, of course, exempt.
+### Contributing
 
-##### Config Variables
-All global cslib config variables should start with 'cslib_' and be as short and informatively named as possible.
+Every contribution is valid. Did you use one of the modules? Let us know! You decided not to use it? Tell us why, was it difficult to understand/use? Or there wasn't anything you found useful?
 
-##### Scenes
-Public scenes (modules) should be named `cslib_modulename` (all lowercase) and avoid numbers or special characters (aside from underscores to separate words). All `test` scenes should be named as: `test_modulename`.
\ No newline at end of file
+Do you have a routine in mind you believe would be useful? Shoot it! Or do you want to try it yourself and contribute with code? Great! Just make sure to read the [contributing guidelines](./CONTRIBUTING.md) for the code style conventions and other more technical stuff.
+
+The idea is for the project to be driven by the community, so everyone is welcome.
+
```

Comment:
Should we include a Roadmap section with things planned for the future?

Reply:
Do we have a roadmap?  :-)

Reply:
Not yet? 😬😂